Header header
uint8 mode
bool GNSSFix
bool compassAiding
bool compassActive